src/Cake.Common/Tools/Roundhouse/RoundhouseRunner.cs

```diff
@@ -156,7 +156,7 @@ protected override string GetToolName()
         /// <returns>The tool executable name.</returns>
         protected override IEnumerable<string> GetToolExecutableNames()
         {
-            return new[] { "rh.exe" };
+            return new[] { "rh.exe", "dotnet-roundhouse", "dotnet-roundhouse.exe" };
```

Comment:
use expression body?

Reply:
Not sure if it is justified.  What is here is in keeping with other PR's for similar changes:

https://github.com/cake-build/cake/pull/2418
https://github.com/cake-build/cake/pull/2388